tests/mock_network/utils.rs

```diff
@@ -191,7 +191,7 @@ impl<'a> TestNodeBuilder<'a> {
     }
 
     pub fn create(self) -> TestNode {
-        let (inner, user_event_rx) = self
+        let (inner, user_event_rx, _client_rx) = self
```

Comment:
Is this correct, perhaps only naming. should they be client_rx and node_rx or similar? (just user and client seems quite similar in name)

Reply:
They are different channel, user_event is the channel where routing send event (before it also contained the client message). client_rx is the new channel that contains message for client directly from quic-p2p (which we don't care in our tests since we don't do client stuff in routing.